magit-todos.el

```diff
@@ -359,6 +359,12 @@ from the \"topic2\" branch, this option could be set to
 \"topic\"."
   :type 'string)
 
+(defcustom magit-todos-trim-filename-fn nil
+  "Apply function to file name to change its representation.
+Function takes relative file path as an argument and must return string."
+  :type '(choice (const :tag "Identity" nil)
```

Comment:
The tag for `nil` should be something like `Don't trim`.

Reply:
Done